packages/shared/lib/core/ledger/interfaces/ledger.interface.ts

```diff
@@ -1,6 +1,6 @@
-import { IEvmTransactionData } from '@core/layer-2/interfaces'
+import type { TxData } from '@ethereumjs/tx'
 
 export interface ILedger {
     generateEvmAddress(bip32Path: string, verify?: boolean): void
-    signEvmTransaction(data: IEvmTransactionData, bip32Path: string): void
+    signEvmTransaction(data: TxData, bip32Path: string): void
```

Comment:
Would be good to create a wrapper type around this so we can help slightly with coupling to a particular library - if need to swap the underlying type out it will be easier with the wrapper